Demo/Components/BottomSheet/BottomSheetMechanicsDemoViewController.swift

```diff
@@ -12,7 +12,14 @@ protocol RootViewControllerDelegate: class {
 
 class RootViewController: UIViewController {
 
+    // MARK: - Public properties
+
     weak var delegate: RootViewControllerDelegate?
+    lazy var draggableLabelFrame = CGRect(origin: CGPoint(x: 0, y: 150), size: CGSize(width: view.frame.width, height: 44))
```

Comment:
Could it be that `view.frame` is not set yet when we access `draggableLabelFrame`?

Reply:
I'm not completely sure, but I think it always will be as long as it happens after we've called `init`.

This is from the documentation for [`UIViewController.loadView()`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621454-loadview):
> The view controller calls this method when its view property is requested but is currently nil. This method loads or creates a view and assigns it to the view property.

Reply:
It will be set after init, but it might not be the size we want. I guess we can only be sure the frame is correct after layoutSubviews is called when using Auto Layout.

Reply:
Yeah, what @shredlocker said. By "not set" I meant zero or not correct frame that you might get in `viewDidLoad` for example. How about making `draggableLabelFrame` a computed var, then at least you always get the current view width:

```swift
var draggableLabelFrame: CGRect { 
    return CGRect(
        origin: CGPoint(x: 0, y: 150), 
        size: CGSize(width: view.frame.width, height: 44)
    )
}
```

Reply:
Yeah, that makes sense 👍 I'll change it